server/configs/included_files.sig

```diff
@@ -0,0 +1,4 @@
+tVKrG/8xsL04ugE/PSmA0XL6vxHwac2poCPDIrlKrqsxVlipuK+++ZGguudqRf35uPYJ6q7coCH8yBf5xH5BBA== included_files.conf
```

Comment:
Feels like there needs to be a tighter coupling to the file name, should we include path? Also should the filename be first?